app/controllers/Tournament.scala

```diff
@@ -302,4 +319,19 @@ object Tournament extends LilaController {
     },
     expireAfter = _.ExpireAfterWrite(15.seconds)
   )
+
+  private val teamApi = lila.app.Env.team.api
+  private def teamIdsIBelongTo(me: lila.user.User): Fu[List[String]] =
+    teamApi.mine(me) map { teams =>
```

Comment:
fetching entire team objects to only keep the id is wasteful.

I also notice an existing cache of team ids by user: https://github.com/ornicar/lila/blob/ab6f131f44e03b2e0b97f85d802ec16ac21b97a7/modules/team/src/main/Cached.scala#L22

`Env.team.cached.teamIdsList` should do.